dapps/digital-twin-detail/src/components/container/DataSetForm.ts

```diff
@@ -201,4 +220,57 @@ export default class DataSetFormComponent extends mixins(EvanFormComponent) {
       ? formData.primitive
       : formData;
   }
+
+  /**
+   * Sets the control validation.
+   *
+   * @param      {EvanFormControl}  control    The control
+   * @param      {string}           type       The type
+   * @param      {any}              subSchema  The sub schema
+   */
+  setControlValidation(controlOpts: EvanFormControlOptions, type: string, subSchema: any): void {
+    // check if min value is set required flag
+    if (type === 'array' || type === 'object' || type === 'boolean'
+      || (ajvMinProperties[type] && !Number.isNaN(subSchema[ajvMinProperties[type]]))) {
+      controlOpts.uiSpecs.attr.required = true; // eslint-disable-line no-param-reassign
```

Comment:
Create a local variable in this simple case instead of disabling the rule?
```
const options = controlOpts;
```

If you want, we could have a discussion on that rule in general.